README.md

```diff
@@ -1,16 +1,164 @@
+
+Please see [Getting Started](https://github.com/microsoft/vcpkg#getting-started) on how to use Vcpkg package manager with your application. In summary,
+
+If you use Linux or Mac:
+
+```sh
+git clone https://github.com/microsoft/vcpkg
+./vcpkg/bootstrap-vcpkg.sh
+./vcpkg/vcpkg install "hazelcast-cpp-client[openssl]" --recurse
+``` 
+
+If you use Windows:
+
+```bat
+git clone https://github.com/microsoft/vcpkg
+.\vcpkg\bootstrap-vcpkg.bat
+.\vcpkg\vcpkg install "hazelcast-cpp-client[openssl]:x64-windows" --recurse
+``` 
+The above code snippet will install `hazelcast-cpp-client` with its `boost` dependencies.
+
+After the installation, the library is available for usage. For example, if you are using CMake for your builds, you can use the following cmake build command with the `CMAKE_TOOLCHAIN_FILE` cmake option to be the `vcpkg.cmake`.
+```bat
+> cmake -B [build directory] -S . -DCMAKE_TOOLCHAIN_FILE=[path to vcpkg]/scripts/buildsystems/vcpkg.cmake
+> cmake --build [build directory]
+```
+
+The above code will install `hazelcast-cpp-client` with its `boost` and `openssl` dependencies.
+
+You can find more details on using a Vcpkg installed package from different IDEs in your projects from the [Vcpkg Official Getting Started](https://github.com/microsoft/vcpkg#getting-started) documentation.
+
+##### Other Methods
+
+You can also install the hazelcast-cpp-client with Conan and from source code. You can more information from [Reference Manual](https://github.com/akeles85/hazelcast-cpp-client/blob/readme_update/Reference_Manual.md#11-installing).
+
+## Overview
+
+### Usage
+
+There is an example project in sample_project directory. You can run the example as below:
+
+If you use Linux or Mac:
+
+```sh
+cmake -B build -S . -DCMAKE_TOOLCHAIN_FILE=[path to vcpkg]/scripts/buildsystems/vcpkg.cmake &&
+cmake --build build &&
+./build/client
+```
+
+If you use Windows:
+
+```bat
+cmake -B build -S . -DCMAKE_TOOLCHAIN_FILE=[path to vcpkg]\scripts\buildsystems\vcpkg.cmake && 
+cmake --build build && 
+.\build\Debug\client
+```
+
+The sample code creates a client, the client automatically connects to the cluster.
+It creates a map named "personel_map" and puts the records inside it.
+It then gets all the entries from the cluster and prints them.
+```c++
+#include <hazelcast/client/hazelcast_client.h>
+int main() {
+    auto hz = hazelcast::new_client().get(); // Connects to the cluster
+
+    auto personel = hz.get_map("personel_map").get();
+    personel->put<std::string, std::string>("Alice", "IT").get();
+    personel->put<std::string, std::string>("Bob", "IT").get();
+    personel->put<std::string, std::string>("Clark", "IT").get();
+    std::cout << "Added IT personel. Logging all known personel" << std::endl;
+    for (const auto &entry : personel->entry_set<std::string, std::string>().get()) {
+        std::cout << entry.first << " is in " << entry.second << " department." << std::endl;
+    }
+    
+    return 0;
```

Comment:
`personel` should be renamed to `personnel`

Reply:
Fixed